packages/wonder-blocks-testing/src/gql/mock-gql-fetch.ts

```diff
@@ -9,6 +9,8 @@ export const mockGqlFetch = (): GqlFetchMockFn =>
     mockRequester<GqlMockOperation<any, any, any>, any>(
         gqlRequestMatchesMock,
         // Note that the identation at the start of each line is important.
+        // TODO(somewhatabstract): Make a stringify that indents each line of
+        // the output properly too.
```

Comment:
**note:** Would love to fix the stringified bits long term too for a nicer output. Not in scope for this change though